atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/ResultsExtractor.java

```diff
@@ -91,4 +100,9 @@ public abstract void internalExtractResult(long startTs,
                                                long ts);
 
     public abstract Map<Cell, T> asMap();
+
+    protected Meter getPostFilteredCellsMeter(Class clazz) {
+        // TODO(hsaraogi): add table names as a tag
+        return metricsManager.registerMeter(clazz, "startTsCellFilterCount");
```

Comment:
@schlosna should we be prefixing these meters with `com.palantir.atlasdb`?

Reply:
For internal type stats, I think we're continuing to use something like the fully qualified classname where metrics originate, so in this case `clazz` should get us that.

@ellisjoe for SA

Reply:
also, is `startTsCellFilterCount` the right name here?

Reply:
Added a test to ensure that the FQCN is registered.

Reply:
@schlosna I think you are worried about the name due to the method name, the idea here is that cells read from the database are being postfiltered based on the startTs w.r.t the transactionTs before being returned to the client. We want to record the count for the cells read but discarded here. Do you have a name suggestion?